Header header
uint32 num_boxes
float32[] box_vertices
string[] object
float32[] score